src/__tests__/__snapshots__/theme-provider.js.snap

```diff
@@ -127,11 +127,11 @@ exports[`with theme prop of padding 20px 1`] = `
       }
     }
   >
-    <glamorous(div)>
+    <glamorous.div>
       <div
         className="css-1e41o48"
       />
-    </glamorous(div)>
+    </glamorous.div>
```

Comment:
The new name is more intuitive, but is it worth major version?
This change makes no difference to people who use awesome displayname plugin which provides much better debugging experience.